packages/editor-ui/src/constants.ts

```diff
@@ -654,14 +654,10 @@ export const UPDATE_WEBHOOK_ID_NODE_TYPES = [FORM_TRIGGER_NODE_TYPE];
 
 export const CREATOR_HUB_URL = 'https://creators.n8n.io/hub';
 
-/**
- * Units of time in milliseconds
- */
+// Units of time in milliseconds
```

Comment:
JSDoc comments are shown in tooltips, regular comments are not, so would be nice if this was kept as JSDoc. Or renamed to something like `InMilliseconds`.

![image](https://github.com/n8n-io/n8n/assets/10324676/4fe33cc3-d83f-4f93-8189-0d4eba252404)


vs

![image](https://github.com/n8n-io/n8n/assets/10324676/9392859e-f57a-4999-9337-32f7fb69c1a1)